orchestrator-salt/src/main/resources/salt/salt/postgresql/init.sls

```diff
@@ -1,6 +1,8 @@
 {%- from 'metadata/settings.sls' import metadata with context %}
 
 {% set configure_remote_db = salt['pillar.get']('postgres:configure_remote_db', 'None') %}
+{% set postgres_data_directory = salt['pillar.get']('postgres:postgres_data_directory') %}
+{% set postgres_data_on_attached_disk = salt['pillar.get']('postgres:postgres_data_on_attached_disk', 'None') %}
```

Comment:
default should be false and not `None`, shouldn't it?

Reply:
Yes, that would answer my next question in the if condition.

Reply:
Default value has set to 'False'